docs/connectors/slack.md

```diff
@@ -97,3 +97,75 @@ class BlocksSkill(Skill):
 ```
 
 ![](https://user-images.githubusercontent.com/1610850/58658951-ac523300-8319-11e9-8c2a-011469a436d0.png)
+
+## Interactive Actions
+
+Slack apps can use [interactive features](https://api.slack.com/interactivity) to achieve much more than just one-way communication. Apps can implement a number of interaction entry points that allow users to intentionally invoke a response from the app.
+
+When one of those entry points is triggered, a new aspect is introduced to the [interaction transaction](https://api.slack.com/interactivity/handling) — the interaction payload. This payload is a bundle of information that explains the context of the user action, giving the app enough to construct a coherent response.
+
+For example, when you click a button in a rich Slack message or use a message action (a todo list app may have an "add to list" action that can be performed on any message) Slack will send an event to a separate webhook endpoint.
+
+### Configure Slack App for Interactive Events
+
+- Open your app's [management dashboard](https://api.slack.com/apps)
+- Click on `Interactive Components` in the sidebar.
+- Toggle the `Interactivity` switch on.
+- Save the HTTPS URL of your bot's slack interactivity endpoint (`/connector/slack/interactions`).
+    - *Example:* `https://slackbot.opsdroid.com/connector/slack/interactions`
```

Comment:
Minor request here, could we use a generic domain rather than suggest we may provide some hosting or domain services.

```suggestion
    - *Example:* `https://slackbot.example.com/connector/slack/interactions`
```

Reply:
Done.